src/less/overwrites/_type.less

```diff
@@ -1,5 +1,22 @@
+
+h1{
```

Comment:
You are lacking the .h1 classes from bootstrap ( and all following). Please have a look at the typefile of bootstrap